bridge_adaptivity/module/templates/module/collection_detail.html

```diff
@@ -10,7 +10,7 @@
 
 {% block content %}
   {% include "module/modals/modal_wrapper.html" %}
-  {%  url 'module:activity-add' collection_slug=collection.slug as activity_add %}
+  {%  url 'module:activity-add' collection_id=collection.id as activity_add %}
```

Comment:
here and everywhere exchanging collection_slug by collection_id should be reverted.